pkg/storer/reserve.go

```diff
@@ -297,6 +297,9 @@ func (db *DB) ReservePutter() storage.Putter {
 	return putterWithMetrics{
 		storage.PutterFunc(
 			func(ctx context.Context, chunk swarm.Chunk) error {
+				if db.reserve == nil {
```

Comment:
I am not sure that this is enough, I think that the errors says that `chunk` or `chunk.Stamp()` could be also nil

Reply:
It's the reserve that is nil